app/views/dashboard/form/shared/_action_footer.html.erb

```diff
@@ -1,7 +1,8 @@
 <% primary_action = local_assigns.fetch(:primary_action, :save_and_continue) %>
+<% button_key = current_user.admin? ? :admin_save : param_key %>
```

Comment:
How about we call this `secondary_action` instead of `button_key` since there are three buttons in this footer, and this will match up better with `primary_action` above?

Reply:
Yep, that was my first choice, but I named it button_key because it didn't follow the _exact_ same pattern with primary_action, but that doesn't matter.... `secondary_action` makes more sense.